### PR TITLE
Explain how to put env vars inside of app with example of google maps api key

### PR DESCRIPTION
**Pull request by BondarenkoStas**
Regarding https://github.com/NevercodeHQ/codemagic-support/issues/82

That's the shortest approach I found, but adding Priit to review as maybe he knows better approaches.
Tested on this app https://github.com/NevercodeHQ/triage/tree/master/flutter_plugin_examples/google_maps_flutter_example, it worked

### CODE REVIEW

File: content/building/environment-variables.md [hunk 1]
```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
 4. Click **Save** to save the settings.
 
 ![](../uploads/env_vars.PNG)
+
+## Accessing environment variables from your application
+
+The next examples show how to place your Google Maps api key into an Android or iOS application from an environment variable. Doing it allows you to not store your secret key in the source code.
```
Comment (priitlatt) [inline content/building/environment-variables.md:84]:
```suggestion
The next examples show how to place your Google Maps API key into an Android or iOS application from an environment variable. Doing it allows you not to store your secret key in the source code.
```

Comment (BondarenkoStas) [reply content/building/environment-variables.md]: fixed in 
18937f0

File: content/building/environment-variables.md [hunk 2]
```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
 4. Click **Save** to save the settings.
 
 ![](../uploads/env_vars.PNG)
+
+## Accessing environment variables from your application
+
+The next examples show how to place your Google Maps API key into an Android or iOS application from an environment variable. Doing it allows you not to store your secret key in the source code.
```
Comment (helinanever) [inline content/building/environment-variables.md:84]: the next > the following..
Doing it allows you not to... > With this approach you will not have to store your secret key in the repository.

File: content/building/environment-variables.md [hunk 3]
```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
...
+
+1. Add your key as an environment variable **MAPS_API_KEY**
+
```
Comment (helinanever) [inline content/building/environment-variables.md:87]: ...as an environment variable with the name ..
NB: environment variable names should be formatted with upticks ` `

File: content/building/environment-variables.md [hunk 4]
```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
...
+### Android
+
+2. Read your key from an environment variable to **build.gradle**
```
Comment (helinanever) [inline content/building/environment-variables.md:90]: Read your key > Read the key...

File: content/building/environment-variables.md [hunk 5]
```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
...
+
+        defaultConfig {
+            // Other values set here
+            resValue "string", "maps_api_key", "$System.env.MAPS_API_KEY"
+        }
+
+3. Read your key from a **build.gradle** value to **AndroidManifest.xml**
```
Comment (helinanever) [inline content/building/environment-variables.md:97]: Read your key > Read the key from the **build.gradle** value...

File: content/building/environment-variables.md [hunk 6]
```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
...
+
+        <meta-data android:name="com.google.android.geo.API_KEY"
+            android:value="@string/maps_api_key"/>
+
+### iOS (Swift)
+
+2. Read your key from an environment variable to **Info.plist**
```
Comment (helinanever) [inline content/building/environment-variables.md:104]: Read your key > Read the key from the environment variable..

File: content/building/environment-variables.md [hunk 7]
```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
...
+
+        <key>MAPS_API_KEY</key>
+        <string>$(MAPS_API_KEY)</string>
+
+3. Read your key from an **Info.plist** value to **AppDelegate.swift**
```
Comment (helinanever) [inline content/building/environment-variables.md:109]: Read your key > Read the key from the...

File: content/building/environment-variables.md [hunk 8]
```diff
@@ -78,3 +78,40 @@ Flutter projects can be configured both in .yaml file as well as in the Codemagi
...
+
+        GMSServices.provideAPIKey(Bundle.main.object(forInfoDictionaryKey: "MAPS_API_KEY") as? String ?? "")
+
+### iOS (Objective-C)
+
+2.  Read your key from an environemnt variable to **AppDelegate.m** as in the [example](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter/example/ios/Runner/AppDelegate.m).
```
Comment (helinanever) [inline content/building/environment-variables.md:115]: Read your key from the environment variable to..

File: content/building/environment-variables.md (hunk 5 above)
Comment (priitlatt) [reply content/building/environment-variables.md:97]: Use inline code (backticks) format for file names?

**Comment by BondarenkoStas** [comment]
fixed comments, used `` to wrap file names instead of doing them bold

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**